.gitignore

```diff
@@ -1,15 +1,11 @@
-# Ignore Play! working directory #
-bin/
-/db
-.eclipse
-/lib/
-/logs/
-/modules
-/project/target
-/target
-tmp/
-test-result
-server.pid
-*.eml
-/dist/
-.cache
+
+target/
+*.ipr
+*.iws
```

Comment:
.ipr and .iws are the intellej auto-generated files which contains project files related information. To make it simple and clean, except source files remaining all are excluded and also unused statements from .gitigore removed.